cmd/nuke.go

```diff
@@ -156,11 +156,13 @@ func (n *Nuke) Scan() error {
 	for _, regionName := range n.Config.Regions {
 		region := NewRegion(regionName, n.Account.ResourceTypeToServiceType, n.Account.NewSession)
 
+		var featureFlags = n.Config.FeatureFlags
```

Comment:
Was unsure of how best to load the feature flags into a variable (maybe global scope?) so just decided to put it here